conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalytics.java

```diff
@@ -289,11 +253,24 @@ public void resumeTracking() {
     }
     // endregion
 
-    private void customEvent(BitmovinPlayerEvent event) {
+    private void customEvent(PlayerEvent event) {
```

Comment:
I'm not 100% sure but I think there should be a super type `Event` which could be used here or?

Reply:
Yes there is the `Event` super type. Would reduce duplicated code for `PlayerEvent` and `SourceEvent`

Reply:
updated